scripts/bundleConfig.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const config = require('../shared/data/constants.js')
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args))
+
+const outputEtagFile = './shared/data/etags.json'
+const bundlePath = './shared/data/bundled/'
+const configProps = config.tdsLists.find((c) => c.name == 'config');
+const etags = {};
+
+async function getConfig(name, url) {
+    let response = await fetch(url)
+    if (!response.ok) {
+        throw new Error(`Failed to load tds list ${name}: ${url}`)
+    }
+    etags[`${name}-etag`] = response.headers.get('etag')
+    const fileText = await response.text()
+    const urlObject = new URL(url)
+    const fileName = urlObject.pathname.split('/').pop()
+    fs.writeFileSync(`${bundlePath}/${fileName}`, fileText)
```

Comment:
can we do some sanity check before writting? Like verify if this is valid JSON and throw if not? Throwing will fail the build process right?

Reply:
Yeah it will fail it, for now I've just added JSON parse which is a good enough sanity check. We could additionally run the tests on the file but that's a much bigger change.